### ISSUE TITLE
404s loading .json files

### ISSUE THREAD

**Issue by simonihmig** [ember-learn/cli-guides#55]
Seems the `index` pages all trigger a request to a non-existing .json file:

![image](https://user-images.githubusercontent.com/1325249/49046766-85ce0680-f1d5-11e8-9902-18cf264c224f.png)

**Comment by mansona** [comment]
@simonihmig you are correct in saying there are performance costs, as there will be a double round-trip in these cases. 

The architecture of the Ember Guides App (and all this tech is just that same app in a slightly more generalised way) vastly favoured **simplicity** so that we could encourage as many contributions as possible.

We are at the point now that it is likely that we can spend some cycles coming up with a solution to this issue but for now I think it has a relatively low priority 🤔 I could be wrong. If you wanted to pair on figuring it out we could live-stream the investigation 😉

**Comment by mehulkar** [comment]
@simonihmig this is mostly not happening anymore as far as a I can see. Good to close? I still see `version.json` 404'ing, but I'm guessing that's a separate issue

**Comment by simonihmig** [comment]
@mehulkar thanks for the reminder. I just did test it our again, but still there are some 404 coming up. It seems this is still the case for every first page of a new chapter (basic use, advanced, addons etc.):

![image](https://user-images.githubusercontent.com/1325249/78345409-ea555e80-759d-11ea-8711-0d208366ec67.png)

**Comment by mansona** [comment]
This just requires an update to guidemaker for it to be fixed 👍 it was fixed in a bugfix upstream

You might have to update guidemaker-ember-template at the same time 🤔 but you **shouldn't** 😂
